source/_integrations/nuki.markdown

```diff
@@ -24,9 +24,26 @@ The Nuki integration allows you to control [Nuki Smart Locks](https://nuki.io/en
 ## Prerequisites
 
 To add a Nuki bridge to your installation, you need to enable developer mode on your bridge and define a port and an access token. This can be achieved using the [Android app](https://play.google.com/store/apps/details?id=io.nuki) or [iPhone app](https://apps.apple.com/app/nuki-smart-lock/id1044998081). Go to manage my devices, and select the bridge. Within the bridge configuration turn on the HTTP API and check the details in the screen. Please note that the API token should be 6-20 characters long, even though the app allows you to set a longer one.
+For faster updates you can configure to use the callback function of the Nuki bridge. This requires your Home Assistant to be reachable via HTTP by the Nuki bridge. If required, you can set a custom URL for the callback, which allows HTTP traffic to `/api/nuki/callback/{entry-id}`.
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+  Host:
+    description: Hostname or IP address of your Nuki bridge, e.g., `192.168.1.25`.
+  Port:
+    description: Port of the Nuki bridge HTTP API.
+    default: 8080
+  Token:
+    description: Token to authenticate against the Nuki bridge HTTP API.
+  "Enable Nuki callback":
+    description: Enable the callback function of the Nuki bridge HTTP API.
+    default: false
+  "Override Home Assistant URL":
+    description: Override the base URL of the callback. Leave empty to use the Local Network Address of Home Assistant. 
```

Comment:
This control should not be needed. Home Assistant provides configuration for this already.

Reply:
I included this, as I imagine a lot of people also have an https configured for the internal Home Assistant URL. As this needs to be http, you could configure a seperate url. Or should we just take the internal URL and change it to http always?

Reply:
>  included this, as I imagine a lot of people also have an https configured for the internal Home Assistant URL

That is up to them, and not a concern each integration should try to resolve.

> Or should we just take the internal URL and change it to http always?

Not at all, that is making even more assumptions.

Reply:
Okay, should I then add a check to print out a log entry if it is set to https, because it cannot be added to the bridge?

Reply:
You could raise an issue in the repair center when an invalid combination is detected?

Reply:
That's a good idea. So would you prefer to have the callback choice removed, or just defaulted to true?

Reply:
We generally prefer the lesser options route.

Reply:
removed the two options and reworded a bit.